tests/test_reqctx.py

```diff
@@ -156,7 +158,9 @@ def test_greenlet_context_copying(self, app, client):
 
         @app.route('/')
         def index():
+            flask.session['fizz'] = 'buzz'
             reqctx = flask._request_ctx_stack.top.copy()
+            reqctx.session = deepcopy(flask.session)
```

Comment:
i don't think this should be in the test.

Reply:
Isn't the purpose of `test_greenlet_context_copying` to test the raw context copying functionality, separately from `test_greenlet_context_copying_api`? Without that line, the test fails. I can remove the session test from that altogether, but in that case I'm not sure what `test_greenlet_context_copying` is testing for.

Reply:
ah I didn't realize this is some greenlet stuff. nevermind then i guess.

but in that case we're missing a testcase for your feature in a normal wsgi context

Reply:
Hmm, in that case shouldn't there already be a test for `copy_current_request_context` outside of the greenlet testing? This is just a small modification to that existing feature.

Reply:
yes... probably. i hoped we'd have such a test. :/

Reply:
I can write one but it seems a bit outside the scope of this change in that case - I'm not actually certain what that test should look like, because the feature doesn't really do anything useful when used directly inline without a greenlet/thread. I can write one anyway but that might turn this PR into something else :)

Reply:
agreed, let's keep this pr as it is